ts/session/utils/job_runners/jobs/ConfigurationSyncJob.ts

```diff
@@ -208,6 +209,29 @@ class ConfigurationSyncJob extends PersistedJob<ConfigurationSyncPersistedData>
         };
       });
 
+      if (window.sessionFeatureFlags.debug.debugLibsessionDumps) {
+        for (let index = 0; index < LibSessionUtil.requiredUserVariants.length; index++) {
+          const variant = LibSessionUtil.requiredUserVariants[index];
+
+          window.log.info(
```

Comment:
Should this not be window.log.debug? I forget how this works?

Reply:
this is only shown when the debugLibSessionDumps is true, so not on by default already.
It would be weird to have to turn the debug and the debugLibSessionDumps flag to get those printed, I think